dnf/cli/output.py

```diff
@@ -1474,12 +1474,12 @@ def historyListCmd(self, extcmds):
                 elif old.return_code:
                     rmark = lmark = '#'
                     # We don't check .errors, because return_code will be non-0
-                elif old.output:
+                elif old.is_output:
```

Comment:
Output itself can be obtained using `SwdbTrans` method `.output()` - or `dnf_swdb_trans_output (DnfSwdbTrans *)`